docs/manual.rst

```diff
@@ -1,19 +1,28 @@
-
 =======================
 Running pytest-workflow
 =======================
-Run ``pytest`` from an environment with pytest-workflow installed.
+Run ``pytest`` from an environment with pytest-workflow installed or
+``python3 -m pytest`` if using a system-wide or user-wide installation.
 Pytest will automatically gather files in the ``tests`` directory starting with
 ``test`` and ending in ``.yaml`` or ``.yml``.
 
 The workflows are run automatically. Each workflow gets its own temporary
 directory to run. These directories are cleaned up after the tests are
 completed. If you wish to inspect the output of a failing workflow you can use
-the ``--keep-workflow-wd`` flag to disable cleanup.
+the ``--keep-workflow-wd`` flag to disable cleanup. This will also make sure
```

Comment:
Perhaps it would be a good idea to have a small line about where the temporary directory can be found.